test/module_6/webDriverJS.spec.js

```diff
@@ -0,0 +1,31 @@
+const { describe, afterEach, it } = require("mocha");
+const { expect } = require("chai");
+const {Builder, By, until} = require("selenium-webdriver");
+const driver = new Builder().forBrowser("firefox").build();
+
+describe("Selenium-webdriverJS test", function () {
+  afterEach(async () => await driver.quit());
+
+  it("oz.by simple test: find a book, add to cart, delete from cart", async () => {
+      await driver.get("https://oz.by/")
```

Comment:
fyi: you can use `return` instead of `await`

Reply:
thanks